examples/tutorial/tutorial_notebooks/introduction/exercise_1.ipynb

```diff
@@ -78,6 +78,8 @@
     "UserWarning: Distributing <class 'numpy.ndarray'> object. This may take some time.\n",
     "```\n",
     "\n",
+    "Multithreading is implemented using Ray, OmniSci or Dask. On default using Ray.\n",
```

Comment:
I think it would be good to put notebooks in separate locations with appropriate env files (pandas_on_ray/dask, omnisci_on_native).

Reply:
@Rubtsowa, let's put notebooks related to OmniSci usage with respective env file in `modin/examples/tutorial/tutorial_notebooks/introduction/omnisci_on_native`. Also, could you open an issue to refactor `examples/tutorial` for separating notebooks of pandas_on_ray usage into own directory?

Reply:
I created folders ``pandas_on_ray`` and ``omnisci_on_native``

Reply:
Let's leave `pandas_on_ray` stuff as it was before. We should separate it as part of a separate issue.